impl/client/src/main/javascript/web/pentaho/visual/config/vizApi.conf.js

```diff
@@ -639,6 +692,22 @@ define([
         }
       },
 
+      // Donut/Pie
```

Comment:
This rule is not needed as Donut is a subtype of Pie and Pie already has the titlePosition : bottom in the above rule.